### ISSUE TITLE
Create image uploader backend

### ISSUE THREAD

**Issue by RichAWarren** [fac-cycle/fac-cycle#67]

**Comment by mcelearr** [comment]
@gabrielperales has built an example that seems to work so I'm quite happy to take this on today or show people how it works.

**Comment by gabrielperales** [comment]
Let me know if I can help doing something.  The code i did should be inside the route POST /item and calling createItem helper,  storing there the response image url given by cloudinary

**Comment by mcelearr** [comment]
Yeah I was checking out this morning and it was working for me. So we just need to work on integrating it.
